integration-tests/tests/aws/resources/dynamodb-helper.ts

```diff
@@ -2,30 +2,65 @@ import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
 import {
   DeleteCommand,
   DynamoDBDocumentClient,
+  DynamoDBDocumentClientResolvedConfig,
   PutCommand,
+  ServiceInputTypes,
+  ServiceOutputTypes,
 } from "@aws-sdk/lib-dynamodb";
+import { Command } from "@aws-sdk/types";
+
+type DynamoDBDocumentClientCommand<
+  Input extends ServiceInputTypes,
+  Output extends ServiceOutputTypes
+> = Command<
+  ServiceInputTypes,
+  Input,
+  ServiceOutputTypes,
+  Output,
+  DynamoDBDocumentClientResolvedConfig
+>;
+
+type Keys = Record<string, unknown>;
+type TableRecords = { tableName: string; items: Keys };
 
 const client = new DynamoDBClient({ region: process.env.AWS_REGION });
```

Comment:
I don't remember if we implemented adding default env var value in some config file but if it's not there can we add a default value here? Just to be consistent with how the other clients are created.